docs/api.md

```diff
@@ -379,6 +380,21 @@ Or, if you want to allow all domains for example for external auths,
 nativefier https://google.com --internal-urls ".*?"
 ```
 
+#### [block-external-urls]
+
+```
+--block-external-urls
+```
+
+Disable navigation to URLs not considered "internal" (see '--internal-urls'). Instead of opening in an external browser, attempts to navigate to external URLs will be ignored. Default: false
```

Comment:
```suggestion
Forbid navigation to URLs not considered "internal" (see '--internal-urls'). Instead of opening in an external browser, attempts to navigate to external URLs will be blocked, and an error message will be shown. Default: false
```